apps/web/src/components/TrendingSuggestions/TrendingSuggestions.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { render, waitFor, screen } from '@testing-library/react';
+import TrendingSuggestions from './TrendingSuggestions';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import server from '../../mocks/mockServer';
+
+const queryClient = new QueryClient();
+
+const QueriedComponent = () => {
+  return (
+    <QueryClientProvider client={queryClient}>
+      <TrendingSuggestions />
+    </QueryClientProvider>
```

Comment:
Is there a way to abstract this out to our global `render` method?

https://testing-library.com/docs/react-testing-library/setup/

Reply:
Good call